app/models/originality_report.rb

```diff
@@ -0,0 +1,5 @@
+class OriginalityReport < ApplicationRecord
```

Comment:
There are quite a few attributes to keep track of for an originality report, maybe keep it separate? This also lets us associate multiple originality reports with a single submission (which Canvas does) if we need to in the future.